pkg/pillar/cmd/verifier/pubsub/context.go

```diff
@@ -0,0 +1,191 @@
+package pubsub
```

Comment:
Hmm - copyright and license missing in this file.

Reply:
Done